### DISCUSSION TITLE
[Feature] Timestamp range

### DISCUSSION THREAD

**Discussion by max-kibosh** [immich-app/immich#10323] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I'm currently in the process of scanning my family's 35mm film archive.
For many of them, we don't know the exact date of when the images were taken.

It would be amazing if we could have the option to set a date/time range when clicking on "Change date".

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#8266

**Comment by magnuspyke** [comment]
Having the ability to set non-specific dates on photos would be a welcome addition to the product.  This would require decoupling the image timestamp from the estimated date of the photograph.  As a use case, I am working with approximately 10k+ photographs scanned into an archive, many ranging from the late 1800s.  Currently all photographs are time-ordered by the timestamp from when they were scanned.  Being able to organize them according to estimated year/month/etc would ease the pain of sorting through such a large dataset.
